src/components/Description/style.less

```diff
@@ -0,0 +1,55 @@
+@import './../../styles/base.less';
+@sidemargin: 6vw;
+
+.description {
+  padding-top: 5em;
+  margin: 0 @sidemargin;
+  min-height: 100vh;
+  .description-content {
+    display: inline-flex;
+    align-items: center;
+    .description-text-container {
+      width: 60vw - @sidemargin;
+      padding-right: 5vw;
+      .description-text {
+        font-size: 1.5em;
+        padding-bottom: 1em;
+      }
+    }
+    .about-image {
+      padding-bottom: 1em;
+      img {
+        max-width: 30vw - @sidemargin;
+        max-height: 50vh;
+      }
+    }
+  }
+}
+
+@media only screen and (max-width: 1000px) {
+  .about-info {
+    display: block !important;
+    .about-description {
+      width: 100% !important;
+      padding: 0 !important;
+    }
+    .about-image {
+      padding: 0 !important;
+      img {
+        max-width: 80% !important;
+      }
```

Comment:
To avoid using `!important` you move the media query above into the nested styling 

![image](https://user-images.githubusercontent.com/8893880/87898683-bb0b1600-ca03-11ea-9754-21bc7af3ae7a.png)

Reply:
cat is gone now
![image](https://user-images.githubusercontent.com/32044772/89116139-9708f500-d445-11ea-9414-4b14503469e8.png)
~(=^‥^)ノ☆
goodbye cat.